### ISSUE TITLE
Make link/button focus states meet 3:1 contrast requirements

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#6248]
We have a single SASS function that controls focus states. This does not meet the 1.4.11 (WCAG 2.1) 3:1 contrast requirement.
